homeassistant/components/blueprint/schemas.py

```diff
@@ -83,6 +100,7 @@ def is_blueprint_instance_config(config: Any) -> bool:
                     str: vol.Any(
                         None,
                         BLUEPRINT_INPUT_SCHEMA,
+                        BLUEPRINT_INPUT_SECTION_SCHEMA,
```

Comment:
Please add a comment here, or above, explaining that the correct schema is picked by the presence of an `input` dictionary.